code/modules/mob/living/simple_animal/hostile/netherworld.dm

```diff
@@ -0,0 +1,103 @@
sion_far.ogg", "sound/effects/explosion1.ogg", "sound/effects/grillehit.ogg", "sound/effects/genetics.ogg", "sound/effects/heart_beat.ogg", "sound/effects/hyperspace_begin.ogg", "sound/effects/hyperspace_end.ogg", "sound/effects/his_grace_awaken.ogg", "sound/effects/pai_boot.ogg", "sound/effects/phasein.ogg", "sound/effects/picaxe1.ogg", "sound/effects/ratvar_reveal.ogg", "sound/effects/sparks1.ogg", "sound/effects/smoke.ogg", "sound/effects/splat.ogg", "sound/effects/snap.ogg", "sound/effects/tendril_destroyed.ogg", "sound/effects/supermatter.ogg", "sound/misc/desceration-01.ogg", "sound/misc/desceration-02.ogg", "sound/misc/desceration-03.ogg", "sound/misc/bloblarm.ogg", "sound/misc/airraid.ogg", "sound/misc/bang.ogg", "sound/misc/disc.ogg", "sound/misc/highlander.ogg", "sound/misc/interference.ogg", "sound/misc/notice1.ogg", "sound/misc/notice2.ogg", "sound/misc/sadtrombone.ogg", "sound/misc/slip.ogg", "sound/misc/splort.ogg", "sound/weapons/armbomb.ogg", "sound/weapons/beam_sniper.ogg", "sound/weapons/chainsawhit.ogg", "sound/weapons/emitter.ogg", "sound/weapons/emitter2.ogg", "sound/weapon/blade1.ogg", "sound/weapon/bladeslice.ogg", "sound/weapon/blastcannon.ogg", "sound/weapon/blaster.ogg", "sound/weapon/bulletflyby3.ogg", "sound/weapon/circsawhit.ogg", "sound/weapon/cqchit2.ogg", "sound/weapon/drill.ogg", "sound/weapon/genhit1.ogg", "sound/weapon/gunshot_silenced.ogg", "sound/weapon/gunshot2.ogg", "sound/weapon/handcuffs.ogg", "sound/weapon/homerun.ogg", "sound/weapon/kenetic_accel.ogg", "sound/machines/clockcult/steam_woosh.ogg", "sound/machines/fryer/deep_fryer_emerge.ogg", "sound/machines/airlock.ogg", "sound/machines/airlock_alien_prying.ogg", "sound/machines/airlockclose.oggsound/machines/airlockforced.ogg", "sound/machines/airlockopen.ogg", "sound/machines/alarm.ogg", "sound/machines/blender.ogg", "sound/machines/boltsdown.ogg", "sound/machines/boltsup.ogg", "sound/machines/buzz-sigh.ogg", "sound/machines/buzz-two.ogg", "sound/machines/chime.ogg", "sound/machines/cryo_warning.ogg", "sound/machines/defib_charge.ogg", "sound/machines/defib_failed.ogg", "sound/machines/defib_ready.ogg", "sound/machines/defib_zap.ogg", "sound/machines/deniedbeep.ogg", "sound/machines/ding.ogg", "sound/machines/disposalflush.ogg", "sound/machines/door_close.ogg", "sound/machines/door_open.ogg", "sound/machines/engine_alert1.ogg", "sound/machines/engine_alert2.ogg", "sound/machines/hiss.ogg", "sound/machines/honkbot_evil_laugh.ogg", "sound/machines/juicer.ogg", "sound/machines/ping.ogg", "sound/machines/signal.ogg", "sound/machines/synth_no.ogg", "sound/machines/synth_yes.ogg", "sound/machines/terminal_alert.ogg", "sound/machines/triple_beep.ogg", "sound/machines/twobeep.ogg", "sound/machines/ventcrawl.ogg", "sound/machines/warning-buzzer.ogg", "sound/ai/outbreak5.ogg", "sound/ai/outbreak7.ogg", "sound/ai/poweroff.ogg", "sound/ai/radiation.ogg", "sound/ai/shuttlecalled.ogg", "sound/ai/shuttledock.ogg", "sound/ai/shuttlerecalled.ogg", "sound/ai/aimalf.ogg")
```

Comment:
Is this the largest string in our codebase?

Reply:
i didn't want this either

Reply:
Oh my fucking god did you even LISTEN to what you were told to do?
1. ```/static``` list, there need only be a single copy of this per all mi-go.
2. SINGLE QUOTES, ```'``` not DOUBLE ```"``` or else the entire thing is pointless.

Reply:
ummmmmmmmm

Reply:
Hey, at least he tried, y'know?

Reply:
that's it i've had it you're getting a confused emoji under your post